### PR TITLE
Fix GITHUB_OUTPUT on Windows

### PR DESCRIPTION
**Pull request by akuzm**
We have to add it to WSLENV variable or else it's not passed to the WSL processes.

Disable-check: commit-count

### CODE REVIEW

File: .github/workflows/windows-build-and-test.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ jobs:
       run: python .github/gh_config_reader.py
     - name: Set build_type
       id: build_type
+      env:
+        WSLENV: GITHUB_OUTPUT/p
```
Comment (jnidzwetzki) [inline .github/workflows/windows-build-and-test.yaml:29]: For the other actions that use the environment variable `GITHUB_OUTPUT`, we also set `shell: wsl-bash {0}`. Is this intentionally not set here?

**Review by jnidzwetzki — APPROVED** [review]
One minor question, rest LGTM

File: .github/workflows/windows-build-and-test.yaml (hunk 1 above)
Comment (akuzm) [reply .github/workflows/windows-build-and-test.yaml:29]: This is the config action that runs on ubuntu-latest, so bash is the default shell.

Comment (akuzm) [reply .github/workflows/windows-build-and-test.yaml:29]: I guess I have to remove the WSLENV...

**Review by fabriziomello — APPROVED** [review]

File: .github/workflows/linux-32bit-build-and-test.yaml [hunk 2]
```diff
@@ -82,24 +82,33 @@ jobs:
...
+        # wait in case there are in-progress coredumps
+        sleep 10
         if compgen -G "/tmp/core*" > /dev/null; then
           PG_MAJOR=$(echo "${{ matrix.pg }}" | sed -e 's![.].*!!')
           apt-get install postgresql-${PG_MAJOR}-dbgsym >/dev/null
           for file in /tmp/core*
           do
-            gdb /usr/lib/postgresql/${PG_MAJOR}/bin/postgres -c $file <<EOT | tee -a stacktraces.log
-              printf "%s\n\n", debug_query_string
+            gdb /usr/lib/postgresql/${PG_MAJOR}/bin/postgres -c $file <<<"
```
Comment (akuzm) [inline .github/workflows/linux-32bit-build-and-test.yaml:101]: For some reason, heredoc doesn't always work correctly in these scripts.

**Pull request merged**